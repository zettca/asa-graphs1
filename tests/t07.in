70 120
50
50 31
62 65
62 16
62 20
62 28
70 42
70 19
70 13
32 10
32 9
32 18
68 49
68 47
68 67
57 31
57 4
57 66
65 10
65 36
65 26
65 28
33 35
33 30
33 44
42 52
42 41
42 55
42 58
42 38
42 3
39 61
39 27
39 8
39 11
1 69
1 9
1 54
1 11
56 9
56 26
56 18
21 4
21 47
21 18
16 40
16 29
16 12
10 37
10 54
10 51
53 20
53 63
53 26
52 27
2 40
2 63
2 43
2 12
25 27
25 30
40 54
40 28
69 36
69 29
69 49
69 17
23 9
23 61
23 28
36 14
4 64
41 29
41 48
41 46
29 58
29 6
37 44
45 49
9 30
15 63
15 54
15 8
7 51
7 26
7 46
7 12
61 58
61 11
61 35
61 60
59 54
59 5
59 26
59 24
59 44
20 17
20 47
63 6
48 35
48 38
27 43
64 5
17 35
55 43
51 11
51 14
51 67
58 60
6 46
6 24
11 12
35 46
30 22
26 24
18 22
18 44
46 28
60 43
22 67
34 43
