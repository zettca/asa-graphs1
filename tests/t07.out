13
1
1
2
2
3
7
13
15
11
7
3
2
2
